packages/mongodb-chatbot-server/src/test/testConfig.ts

```diff
@@ -28,7 +30,26 @@ import {
   makeFilterNPreviousMessages,
 } from "../processors";
 import { makeDefaultReferenceLinks } from "../processors/makeDefaultReferenceLinks";
+import { MongoMemoryServer } from "mongodb-memory-server";
```

Comment:
How does this interact with some of our Atlas-only functionality, i.e. vector search?

Is the idea that we use the dev db `pages`/`embedded_content` collections for retrieval but store the `conversations` collection in memory?

Reply:
well for the `mongodb-chatbot-server` package, we only really need DB interaction for `conversations`. The core `embedded_content`/vector search logic is in `mongodb-rag-core`. And the `mongodb-chatbot-server` module doesn't directly touch `pages`. 

this PR removes the need for mongodb for `conversations`. We could also in future PR remove need for atlas for `embedded_content` by creating a mock embedded content store or doing something fully in memory like FAISS.

Reply:
made a ticket for mocking embeddedcontentstore in the mongodb-chatbot-server instead of using atlas vector search. also included removing azure openai from the tests there. azure openai causes tests to be quite long running - https://jira.mongodb.org/browse/EAI-572